Compilation error caused by implicit type conversion from String to int

When I do a clean and compile operation, I'm getting a couple of compilation errors:

```
[INFO] Compiling 22 source files to C:\Users\tom\Development\Java\git\ApexUnit\target\classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] \Users\tom\Development\Java\git\ApexUnit\src\main\java\com\sforce\cd\apexUnit\client\connection\ConnectionHandler.java:[118,92] error: incompatible types: String cannot be converted to int
[ERROR] \Users\tom\Development\Java\git\ApexUnit\src\main\java\com\sforce\cd\apexUnit\client\connection\ConnectionHandler.java:[185,91] error: incompatible types: String cannot be converted to int
```

I believe the type of proxyPort property in CommandLineArguments.java should be changed from a String to an Integer, because the value for the proxy port argument passed to the ConnectionConfig.setProxy method is expected to be an int, and String cannot be implicitly converted to int.

--- a/src/main/java/com/sforce/cd/apexUnit/arguments/CommandLineArguments.java
+++ b/src/main/java/com/sforce/cd/apexUnit/arguments/CommandLineArguments.java
@@ -92,7 +92,7 @@ public class CommandLineArguments {
 	@Parameter(names = PROXY_HOST, description = "Proxy host if required for access.", required = false)
 	static private String proxyHost;
 	@Parameter(names = PROXY_PORT, description = "Proxy port if required for access.", validateWith = PositiveIntegerValidator.class, required = false)
-	static private String proxyPort;
+	static private Integer proxyPort;
 	@Parameter(names = HELP, help = true, description = "Displays options available for running this application")
 	static private boolean help;
 
@@ -155,7 +155,7 @@ public class CommandLineArguments {
 		return proxyHost;
 	}
 
-	public static String getProxyPort() {
+	public static Integer getProxyPort() {
 		return proxyPort;
 	}
 

